content/tsColinPrecip.Rmd

```diff
@@ -86,28 +84,22 @@ precipMap <- function(precipData, startDate, endDate){
   precip_breaks <- c(seq(0,80,by = 10), 200)
   
   precipData_cols <- precipData %>% 
-    group_by(state_fullname, county_mapname) %>% 
+    group_by(statename, statecounty) %>% 
     summarize(cumprecip = sum(precipVal)) %>% 
-    mutate(cols = cut(cumprecip, breaks = precip_breaks, labels = cols, right=FALSE)) %>% 
+    mutate(cols = cut(cumprecip, breaks = precip_breaks, labels = cols, right=FALSE)) %>%
     mutate(cols = as.character(cols))
   
   par(mar = c(0,0,3,0))
   
   # png('tsColin.png', width = 7, height = 5, res = 150, units = 'in')
-  m1 <- map('county', regions = precipData_cols$state_fullname, col = "lightgrey")
-  m2 <- map('state', regions = precipData_cols$state_fullname, 
+  m1 <- map('county', regions = precipData_cols$statename, col = "lightgrey")
```

Comment:
I used `m1` to set up all the counties (so if they don't have data in precipData_cols, they still show up) and provides the correct order of counties. Then `m2` outlines the states slightly darker. Then `m3` adds the color to counties w/ data.